manifests/g/Google/Chrome/131.0.6778.140/Google.Chrome.installer.yaml

```diff
@@ -0,0 +1,46 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Google.Chrome
+PackageVersion: 131.0.6778.140
+InstallerLocale: en-US
+InstallerType: msi
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- http
+- https
+- mailto
+- mms
+- tel
+- webcal
+FileExtensions:
+- htm
+- html
+- pdf
+- shtml
+- svg
+- webp
+- xht
+- xhtml
+ProductCode: '{D9596C6B-431E-3638-ACB7-B4B0D24D2D1B}'
+ReleaseDate: 2024-12-10
+AppsAndFeaturesEntries:
+- DisplayName: Google Chrome
+  Publisher: Google LLC
+  DisplayVersion: 70.158.32908

```

Comment:
This looks wrong

Reply:
```
--> Comparing ARP Entries

DisplayName   DisplayVersion Publisher  ProductCode                            Scope
-----------   -------------- ---------  -----------                            -----
Google Chrome 131.0.6778.140 Google LLC {D9596C6B-431E-3638-ACB7-B4B0D24D2D1B} Machine
```

Reply:
Chrome uses a different version format in MSI database from the actual one, explained in https://issues.chromium.org/issues/382215764#comment8

Reply:
Chrome uses a different version format in MSI database from the actual one, explained in https://issues.chromium.org/issues/382215764#comment8

Reply:
https://github.com/russellbanks/Komac/commit/540a620c5e21d5a45d960477528b61b6b759f0dd